src/models/RevAiApiDeploymentConfigConstants.ts

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable no-shadow */
 export enum RevAiApiDeployment {
-    US = 'application/vnd.rev.transcript.v1.0+json',
-    EU = 'text/plain'
+    US = 'US',
+    EU = 'EU'
```

Comment:
i guess why don't we do https://chatgpt.com/share/671ae3b5-5aa8-8004-abcf-cb1511a7dd3e or did i not correctly got your intentions here?

Reply:
this is for node, your example is for python